.github/workflows/flutter.yml

```
@@ -47,6 +47,22 @@ jobs:
           channel: stable
           cache: true
 
+      - name: Import certificates
+        if: ${{ secrets.APPLE_DEV_CERTIFICATE != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)